codelists/actions.py

```diff
@@ -687,20 +688,55 @@ def convert_bnf_codelist_version_to_dmd(version, published=False):
     """
     assert version.coding_system_id == "bnf"
     dmd_csv_data = version.dmd_csv_data_for_download()
-    converted_codelist = create_old_style_codelist(
-        owner=version.codelist.owner,
-        name=f"{version.codelist.name} - dmd",
-        slug=f"{version.codelist.slug}-dmd",
-        coding_system_id="dmd",
-        coding_system_database_alias=most_recent_database_alias("dmd"),
-        methodology=f"Converted from [pseudo-BNF system codelist]({version.get_absolute_url()})",
-        description=version.codelist.description,
-        csv_data=dmd_csv_data,
-        references=[
-            {"text": ref.text, "url": ref.url}
-            for ref in version.codelist.references.all()
-        ],
+    dmd_name = f"{version.codelist.name} - dmd"
```

Comment:
Maybe add to the docstring for the `convert_bnf_codelist_version_to_dmd` method explaining what happens if the handle already exists.